### ISSUE TITLE
odd "Error: Success: (200) OK" maybe json related?

### ISSUE THREAD

**Issue by paul-shannon** [nicolewhite/RNeo4j#47]
First let me say:  a very fine package!  Thanks.
Second: I have been using it, in a learning mode, these last few days, and everything has worked great.  I suspect the odd error I get now (see subject line) may be the result of my getting some package updates (maybe httr_1.1.0?) due to other work I am doing.  sessionInfo() below...

The error:

````
   library(RNeo4j)
   gdb <- startGraph("http://localhost:7474/db/data/", username="neo4j", password="isb");
   Error: Success: (200) OK
````

This stackoverflow post reports a similar-seeming error, though in a quite different context
[stackoverflow post](http://stackoverflow.com/questions/6186770/ajax-request-return-200-ok-but-error-event-is-fired-instead-of-success)

````
sessionInfo()
sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.11.2 (El Capitan)

locale:
[1] C

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] RNeo4j_1.6.2         jsonlite_0.9.19      httr_1.1.0           BiocInstaller_1.20.1

loaded via a namespace (and not attached):
[1] compiler_3.2.3 R6_2.1.2       tools_3.2.3    curl_0.9.5    
```
Many thanks!

Paul Shannon
Institute for Systems Biology

**Comment by nicolewhite** [comment]
Thanks. I imagine `httr` has introduced a slight change in how it returns the response. I'll take a look tonight.

**Comment by nicolewhite** [comment]
It was a breaking change in `httr`. The fix should be available on CRAN under v1.6.3 shortly.

**Comment by paul-shannon** [comment]
Many thanks, Nicole!

 - Paul
